src/cljs/org/broadinstitute/firecloud_ui/common.cljs

```diff
@@ -104,15 +104,17 @@
   (str (config/api-url-root) "/cookie-authed/download/b/" bucket "/o/" object))
 
 (defn gcs-uri->download-url [gcs-uri]
-  (let [matcher (re-find #"gs://([^/]+)/(.+)" gcs-uri)]
-    (when (= 3 (count matcher)) ;; first match will be the whole thing
-      (gcs-object->download-url (matcher 1) (matcher 2)))))
+  (when (string? gcs-uri)
+    (let [matcher (re-find #"gs://([^/]+)/(.+)" gcs-uri)]
+      (when (= 3 (count matcher)) ;; first match will be the whole thing
+        (gcs-object->download-url (matcher 1) (matcher 2))))))
```

Comment:
I know this wasn't your doing originally ... but could gcs-uri->download-url internally call parse-gcs-uri, so we don't have the duplicate regex code?